File: MODUL.6/PRAK602_2310817120010_NurHikmah.c
#include <stdio.h>

int main()
{
    int jumlahRuangan;

    printf("Input\n");
    scanf("%d", &jumlahRuangan);

    int zetsu_putih[jumlahRuangan];
    for (int i = 0; i < jumlahRuangan; i++)
    {
        scanf("%d", &zetsu_putih[i]);
    }

    // Menghitung jumlah zetsu putih setelah membelah diri pada setiap ruangan
    for (int i = 0; i < jumlahRuangan; i++)
    {
        zetsu_putih[i] *= (i + 1);
    }

    printf("\nOutput\n");
    for (int i = 0; i < jumlahRuangan; i++)
    {
        printf("%d ", zetsu_putih[i]);
    }
    return 0;
}

File: MODUL.6/PRAK602_2310817120010_NurHikmah.py
print("Input")
jumlah_ruangan = int(input())
zetsuputih_per_ruangan = list(map(int, input().split()))

# Proses pembelahan zetsu putih
for i in range(jumlah_ruangan):
    zetsuputih_per_ruangan[i] = zetsuputih_per_ruangan[i] * (i + 1)


print("\nOutput")
for zetsu in zetsuputih_per_ruangan:
    print(zetsu, end=" ")
